bbb-lti/grails-app/services/org/bigbluebutton/BigbluebuttonService.groovy

```diff
@@ -370,4 +372,23 @@ class BigbluebuttonService {
         }
         return list;
     }
+	
+    /**
+     * Get the value of all params from idParams (specified in lti-config)
+     * If none are specified, fallback to using resource_link_id and oauth_consumer_key
+     */
+    private String paramsFromConfig(params) {
```

Comment:
We can use a name for tha function that is more descriptive on what is fetching from config

Reply:
Is the new name better? If not, let me know what you think it should be changed to.